cogni-libs/src/main/java/zone/cogni/libs/jena/utils/JenaUtils.java

```diff
@@ -237,28 +237,21 @@ private static RDFReader getReaderByRdfSyntax(Model model, String language) {
 
   private static String getRdfSyntax(org.springframework.core.io.Resource resource) {
     String extension = StringUtils.lowerCase(StringUtils.substringAfterLast(resource.getFilename(), "."));
-    if ("owl".equals(extension) || "rdf".equals(extension)) {
-      // .rdf is extension for application/rdf+xml and is default syntax in Jena
-      // .owl is here for legacy reasons, but it is often RDF XML because it is the Protégé default
-      return null;
-    }
-    if ("nt".equals(extension)) {
-      // .nt is extension for application/n-triples
-      return "N-TRIPLE";
-    }
-    if ("n3".equals(extension)) {
-      // .n3 is extension for text/n3
-      return "N3";
-    }
-    if ("ttl".equals(extension)) {
-      // .ttl is extension for text/turtle
-      return "TURTLE";
-    }
+    Map<String, String> extensionToLanguageCode = getExtensionToLanguageMap();
 
-    // any other extension falls back to RDF XML
-    return null;
+    return extensionToLanguageCode.getOrDefault(extension, null);
   }
 
+  private static Map<String, String> getExtensionToLanguageMap() {
+    Map<String, String> result = new HashMap<>();
+
+    result.put("nt", "N-TRIPLE");
+    result.put("n3", "N3");
+    result.put("ttl", "TURTLE");
+    result.put("jsonld", "JSONLD");
+
+    return result;
+  }
```

Comment:
private static final Map<> in class and init in static initializer (make map synchronized)